### PR TITLE
Issue #406

### PR DESCRIPTION
**Pull request by kirjaamo**
wsrep_applier_FK_failure_retries added to the options table

### CODE REVIEW

**Comment by kirjaamo** [comment]
@byte For some reason, the options table disappeared when I added this parameter. I cannot see anything wrong in the table format. Can you check?

File: galeracluster/source/documentation/mysql-wsrep-options.rst [hunk 1]
```diff
@@ -182,6 +182,40 @@ You can execute the following ``SHOW VARIABLES`` statement to see how this varia
...
+.. csv-table::
+   :class: doc-options
+
+   "Command-line Format", "``--wsrep_applier_FK_failure_retries``"
+   "System Variable", "``wsrep_applier_FK_failure_retries``"
+   "Variable Scope", "Global"
+   "Dynamic Variable", "Yes"
+   "Permitted Values", "Integer"
+   "Default Value", "``1`` "
+   "Initial Version", "MySQL-wsrep 5.7.44"
```
Comment (byte) [inline galeracluster/source/documentation/mysql-wsrep-options.rst:203]: It only appeared in 8.0.35 not 5.7.44 please fix

Comment (kirjaamo) [reply galeracluster/source/documentation/mysql-wsrep-options.rst]: Fixed

**Review by byte — APPROVED** [review]
This is ok to push

**Pull request merged**